Hunting Queries/AzureActivity/AzureBackupsCeased.yaml

```diff
@@ -0,0 +1,64 @@
+id: 8a5f0ed0-9257-46c0-81a5-7a304afd1aea
+name: Azure Backups Ceased
+description: |
+  'This query can be used during threat hunts to identify potentially malicious tampering with backup policies used
+  by Microsoft Recovery Services Vaults/Azure Backup. The query specifically identifies instances where a backup has stopped
+  being performed on a resource that was previously being backed up. The query executes over a 30 days training window to identify
+  existing backups, it will then perform detection over the most recent 3 days. Additionally, in instances where the backed up resource
+  is a VM, the query will join with virtual machine deletion events to determine if backups ceased due to resource deletion.'
+requiredDataConnectors:
+  - connectorId: AzureActivity
+    dataTypes:
+      - AzureActivity
+tactics:
+  - Impact
+relevantTechniques:
+  - T1485
+  - T1490
+query: |
+  // Total time range for the query, includes detection and training data
+  let timeRange = 60d;
```

Comment:
For hunting queries time ranges, we need to support the features of the hunting blade which requires these be in a certain format.  See - https://github.com/Azure/Azure-Sentinel/blob/master/Hunting%20Queries/AzureActivity/Rare_Custom_Script_Extension.yaml - for example usage.

  let starttime = todatetime('{{StartTimeISO}}');
  let endtime = todatetime('{{EndTimeISO}}');
  let Lookback = starttime - 14d;

Reply:
Added